great_expectations/cli/suite.py

```diff
@@ -318,6 +325,44 @@ def suite_new(suite, directory, empty, jupyter, view, batch_kwargs):
         raise e
 
 
+@suite.command(name="delete")
+@click.option("--suite", "-es", default=None, help="Expectation suite name.")
+@click.option(
+    "--directory",
+    "-y",
+    default=None,
+    help="Delete expectation suite.",
+)
+def suite_delete(suite, directory):
+    """Delete specified expectation suite from data_context expectation store.
+
+    Args:
+        expectation_suite_name: The name of the expectation_suite to delete
+
+    Returns:
+        True for Success and False for Failure.
+    """
```

Comment:
```suggestion
"""Delete an expectation suite from the expectation store."""
```
Click uses these as docstrings so they can be really short to match the existing style.

Reply:
WIP, updated